custom-dockerd-entrypoint.sh

```diff
@@ -4,14 +4,6 @@ DOCKER_OPTS="--config-file=/etc/docker/daemon.json"
 $(which dind) dockerd ${DOCKER_OPTS} >/dev/stdout 2>&1 &
 sleep 1
 java -version
-rm -rf docker-fixtures
```

Comment:
docker-fixtures long ago contains the change on this branch, so no need to build it